### PR TITLE
Init Agent project

### PR DESCRIPTION
**Pull request by yu199195**
Init agent project

### CODE REVIEW

File: shardingsphere-agent/pom.xml [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+                        <goals>
+                            <goal>check</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
+
+</project>
```
Comment (dongzl) [inline shardingsphere-agent/pom.xml:292]: need a blank line of end file.

File: shardingsphere-agent/shardingsphere-agent-bootstrap/pom.xml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+                                    <exclude>org.slf4j:*</exclude>
+                                </excludes>
+                            </artifactSet>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
+</project>
```
Comment (dongzl) [inline shardingsphere-agent/shardingsphere-agent-bootstrap/pom.xml:73]: need a blank line of end file.

File: shardingsphere-agent/shardingsphere-agent-core/pom.xml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        <groupId>org.apache.shardingsphere</groupId>
+        <artifactId>shardingsphere-agent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>shardingsphere-agent-core</artifactId>
+
+
+</project>
```
Comment (dongzl) [inline shardingsphere-agent/shardingsphere-agent-core/pom.xml:32]: need a blank line of end file.

File: shardingsphere-agent/shardingsphere-agent-distribution/pom.xml [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+        <groupId>org.apache.shardingsphere</groupId>
+        <artifactId>shardingsphere-agent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>shardingsphere-agent-distribution</artifactId>
+
+
+</project>
```
Comment (dongzl) [inline shardingsphere-agent/shardingsphere-agent-distribution/pom.xml:32]: need a blank line of end file.

**Review by kimmking — APPROVED** [review]

**Pull request merged**